i18n/en.json

```diff
@@ -2699,9 +2699,14 @@
   "marketplace_modal.installing": "Installing...",
   "marketplace_modal.list.configure": "Configure",
   "marketplace_modal.list.Install": "Install",
+  "marketplace_modal.list.local": "Local",
+  "marketplace_modal.list.local.tooltip": "This plugin is not listed in the marketplace but was installed manually.",
   "marketplace_modal.list.try_again": "Try Again",
   "marketplace_modal.list.update": "Update",
-  "marketplace_modal.list.update_available": "Update available: {version}",
+  "marketplace_modal.list.update_available": "Update available:",
+  "marketplace_modal.list.update_confirmation.confirm_button": "Update",
+  "marketplace_modal.list.update_confirmation.message.warning_major_version": "<p class='alert alert-warning'>This update may contain breaking changes.</p>",
```

Comment:
Should the `<p>` tag be in this messages?

Reply:
Indeed, it's expected at the moment, since I'm using a `FormattedHTMLMessage`. It's awkward to compose these sentences otherwise.

Reply:
See my comment above regarding extracting the parent tag.